components/Layout.js

```diff
@@ -119,10 +145,24 @@ export default function Layout(props) {
         {display.fullscreen ? (
           props.children
         ) : (
-          <LayoutContainer>{props.children}</LayoutContainer>
+          <LayoutContainer>
+            {cloneElement(props.children, { openModal, closeModal })}
```

Comment:
I'm not huge on using this, given React's own documentation warns against using it, but it does work for the time being. I would prefer we look into a different option but that can be done separately as I fear this PR is large enough at this point